dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/webdomain/StreamingJsonRoot.java

```diff
@@ -39,11 +40,14 @@
  */
 @Data
 @RequiredArgsConstructor
+@AllArgsConstructor
 @JsonRootName(value = "root", namespace = DxfNamespaces.DXF_2_0)
 public class StreamingJsonRoot<T> {
   private final Pager pager;
 
   private final String wrapperName;
 
   private final FieldFilterParams<T> params;
+
+  private boolean excludeDefaults;
```

Comment:
The possibility of including this value as a field filter was explored but I think this just makes things even more complicated and messy. Keeping them as separate things makes changes easier & the features easier to reason about.